util/log.go

```diff
@@ -3,23 +3,21 @@ package util
 import (
 	"os"
 
-	logging "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/whyrusleeping/go-logging"
+	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/Sirupsen/logrus"
```

Comment:
@rht at this point, you know more about them to make a better decision on the two. I'd say take the path of least resistance if theyre about the same.